src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx

```diff
@@ -143,6 +143,10 @@ public:
     {
     }
 
+    /** Startup the Wi-Fi in test mode
```

Comment:
I think some more explanation would be helpful on this function.

Reply:
done.